### ISSUE TITLE
"EXCEPTION!" while decompiling (not undertale/deltarune)

### ISSUE THREAD

**Issue by Mc-muffin** [krzys-h/UndertaleModTool#69]
Jut got this screen while decompiling another game's code (bytecode 15, GMS 1.4). Maybe related to #27 (the code seem to contain many arrays) I'll attach the disassembly from the program: 
[Disassembly.txt](https://github.com/krzys-h/UndertaleModTool/files/2580695/Disassembly.txt)

```
EXCEPTION!
System.InvalidOperationException: Stack empty.
  in System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   in System.Collections.Generic.Stack`1.Pop()
   in UndertaleModLib.Decompiler.Decompiler.DecompileFromBlock(Block block, List`1 tempvars, Stack`1 workQueue) in C:\Users\krzys\Documents\Visual Studio 2017\Projects\UndertaleModTool\UndertaleModLib\Decompiler\Decompiler.cs:line 762
   in UndertaleModLib.Decompiler.Decompiler.DecompileFromBlock(Block block) in C:\Users\krzys\Documents\Visual Studio 2017\Projects\UndertaleModTool\UndertaleModLib\Decompiler\Decompiler.cs:line 916
   in UndertaleModLib.Decompiler.Decompiler.Decompile(UndertaleCode code, UndertaleData data) in C:\Users\krzys\Documents\Visual Studio 2017\Projects\UndertaleModTool\UndertaleModLib\Decompiler\Decompiler.cs:line 1573
   in UndertaleModTool.UndertaleCodeEditor.<>c__DisplayClass11_0.<DecompileCode>b__0() in C:\Users\krzys\Documents\Visual Studio 2017\Projects\UndertaleModTool\UndertaleModTool\Editors\UndertaleCodeEditor.xaml.cs:line 251
```
![image](https://user-images.githubusercontent.com/8714476/48482412-9bf3c400-e7de-11e8-8134-c0bea211b28f.png)

**Comment by colinator27** [comment]
Yeah, it looks like a similar, if not the same, issue. I had to translate the error message, which I guess means "empty stack", which is what it is in English.

**Comment by Mc-muffin** [comment]
I'll check it out later, about the disassembled code, there's supposed to be an attachment called "Disassembly.txt" on my first post.

**Comment by krzys-h** [comment]
Oh, I missed that, sorry!

**Comment by Mc-muffin** [comment]
I confirm the problem persist on the latest release (0.1.22)

**Comment by Mc-muffin** [comment]
Solved as of 0.2.0
